lib/Backends/CPU/CPUBackend.cpp

```diff
@@ -147,3 +147,8 @@ bool CPUBackend::canDoIndexTypeDemotion(
       Kinded::Kind::SparseToDenseMaskNodeKind);
   return fromTy == ElemKind::Int64ITy && toTy == ElemKind::Int32ITy;
 }
+
+#include "cpuObjectRegistry.h"
```

Comment:
OK. This include (and probably the function below it) is an issue for our build system that is not CMake based. Can you try to hide this behind some kind of conditional compilation? This feature should be enabled with a special CMake -D... parameter or something like that. By default, this include should not happen and no information from this include should be used.

Reply:
@opti-mix Can we use the approach described above by wrapping under:
```
#if !FACEBOOK_INTERNAL
...
#endif
```
Isn't FACEBOOK_INTERNAL macro enabled by default in your builds?

Reply:
Yes, FACEBOOK_INTERNAL should work as well I guess.